test/powershell/engine/Remoting/RemoteSession.Basic.Tests.ps1

```diff
@@ -1,5 +1,6 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+
```

Comment:
Seems we wanted to fix all test files.

Reply:
Yes, I only fixed this one because I was in it trying to figure out why the tests were failing.  We should do the rest later.